utils/lite-components-testkit/src/main/scala/pl/touk/nussknacker/engine/lite/util/test/LiteKafkaTestScenarioRunner.scala

```diff
@@ -0,0 +1,63 @@
+package pl.touk.nussknacker.engine.lite.util.test
+
+import com.typesafe.config.{Config, ConfigValueFactory}
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.{Deserializer, Serializer}
+import pl.touk.nussknacker.engine.api.component.ComponentDefinition
+import pl.touk.nussknacker.engine.graph.EspProcess
+import pl.touk.nussknacker.engine.lite.util.test.KafkaTestScenarioRunner.{ConsumerRecordWihValue, KafkaInputType, KafkaOutputType, SinkName, SourceName}
+import pl.touk.nussknacker.engine.util.test.TestScenarioRunner
+
+import scala.reflect.ClassTag
+
+object KafkaTestScenarioRunner {
+  type KafkaInputType = ConsumerRecord[String, Any]
+  type KafkaOutputType = ProducerRecord[String, Any]
+
+  val SourceName = "test-kafka-source"
+  val SinkName = "test-kafka-sink"
+
+  implicit class ConsumerRecordWihValue(record: ConsumerRecord[String, Any]) {
+    def withValue(value: Any) = new ConsumerRecord(record.topic, record.partition, record.offset, record.key, value)
+  }
+
+  def createConfig(config: Config): Config =
+    config.withValue("kafka.kafkaAddress", ConfigValueFactory.fromAnyRef("kafka:666"))
+}
+
+class KafkaTestScenarioRunner(components: List[ComponentDefinition], config: Config,
```

Comment:
This question is related with comment above about making all `LiteKafkaComponentProvider` components accessible for tests - if we merge both classes, it will much easier to make it possible. Also we have a plan to merge all sources/sinks to one kafka source and one kafka sink in next Nussknacker release.

Reply:
ok, ignore this comment, let's fucus on others, resolving it